### PR TITLE
Wamoura Family Mobskills

### PR DESCRIPTION
**Pull request by jmcmorris**
**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my PR will be ignored.
- [x] I understand I should leave resolving conversations to the LandSandBoat team so that reviewers won't miss what was said.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have _**tested my code and the things my code has changed**_ since the last commit in the PR and will test after any later commits.

## What does this pull request do?

These changes are focused on improving Wamora mobskills. This primarily involves incorporating data collected by Jimmayus. Two additional changes include

1. Implemented the missing mobskills Erosion Dust and Proboscis
2. Added a new mob group for Wamoura that spawn from eclosion and changed the other Wamoura group to spawn normally. 

## Retail Proof
https://ffxiclopedia.fandom.com/wiki/Category:Wamoura
[Jimmayus' Mob Sheet](https://docs.google.com/spreadsheets/d/1YBoveP-weMdidrirY-vPDzHyxbEI2ryECINlfCnFkLI/edit?gid=57955395#gid=57955395)

## Steps to test these changes

`!gotoid 17027421`
`!exec target:useMobAbility(1951)`
`!exec target:useMobAbility(1952)`
`!exec target:useMobAbility(1953)`
`!exec target:useMobAbility(1954)`
`!exec target:useMobAbility(1955)`

### CODE REVIEW

File: scripts/actions/mobskills/magma_fan.lua [hunk 1]
```diff
@@ -10,7 +10,7 @@ mobskillObject.onMobSkillCheck = function(target, mob, skill)
 end
 
 mobskillObject.onMobWeaponSkill = function(target, mob, skill)
-    local dmgmod = xi.mobskills.mobBreathMove(mob, target, skill, 0.5, 1.25, xi.element.FIRE, 600)
+    local dmgmod = xi.mobskills.mobBreathMove(mob, target, skill, 0.0833, 1, xi.element.FIRE, 600)
```
Comment (Xaver-DaRed) [inline scripts/actions/mobskills/magma_fan.lua:14]: 0.0833 seems a lil weird. I would just put `1 / 12` Just an opinion though, it can stay that way

**Review by Xaver-DaRed — APPROVED** [review]
LGTM

File: scripts/actions/mobskills/magma_fan.lua (hunk 1 above)
Comment (jmcmorris) [reply scripts/actions/mobskills/magma_fan.lua:14]: Yeah, I could go either way. It is one less operation but negligible. Maybe someone else has a stronger opinion on the matter.

Comment (Xaver-DaRed) [reply scripts/actions/mobskills/magma_fan.lua:14]: In my case its more a matter of some1 comming, seeing it and asking "wtf does this number even mean"
As a fraction, on the other hand, seems like a more reasonable number.

But as I said, not really a deal breaker to me.

Comment (claywar) [reply scripts/actions/mobskills/magma_fan.lua:14]: Perhaps a compromise with a comment?  I'm sure its documented somewhere, and could help alleviate future concerns (should one arise)

**Review by claywar — APPROVED** [review]

File: scripts/actions/mobskills/magma_fan.lua (hunk 1 above)
Comment (jmcmorris) [reply scripts/actions/mobskills/magma_fan.lua:14]: added a comment :)

**Pull request merged**